Testscripts/Linux/dpdkUtils.sh

```diff
@@ -306,19 +307,19 @@ function Install_Dpdk () {
 	LogMsg "MLX_PMD flag enabling on ${1}"
 	if type Dpdk_Configure > /dev/null; then
 		echo "Calling testcase provided Dpdk_Configure(1) on ${1}"
-		ssh ${1} "cd ${LIS_HOME}/${DPDK_DIR} && make config T=x86_64-native-linuxapp-gcc"
+		ssh ${1} "cd ${LIS_HOME}/${DPDK_DIR} && make config T=x86_64-native-linuxapp-gcc MAKE_PAUSE=n"
```

Comment:
in future, maybe make will can't be used any more, we need switch build them using meson ?

Reply:
yes. 
I will make changes before 20.11 release.